libvncclient/rfbproto.c

```diff
@@ -2257,6 +2336,8 @@ HandleRFBServerMessage(rfbClient* client)
       }
     }
   }
+  if (!SendIncrementalFramebufferUpdateRequest(client))
```

Comment:
Why did this one move down here?

Reply:
I tested moving this up in its original place and it works. IDK why it was moved down